auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -140,21 +140,30 @@
         <template #item-slot-Status="{ item }">
           <span>{{ status(item) }}</span>
           <!-- Future Effective Icon for PAID_FE status -->
-          <template v-if="status(item) === BusinessState.PAID_FE">
-            <v-tooltip bottom>
-              <template #activator="{ on, attrs }">
-                <v-icon
-                  class="ml-1"
-                  color="#F8661A"
-                  v-bind="attrs"
-                  v-on="on"
-                >
-                  mdi-information-outline
-                </v-icon>
-              </template>
-              <span>This filing will become effective on {{ formatDate(item.effectiveDate) }}</span>
-            </v-tooltip>
-          </template>
+          <IconTooltip
+            v-if="status(item) === BusinessState.PAID_FE"
+            icon="mdi-information-outline"
+            maxWidth="300px"
+            :iconStyling="{'font-size': '1.5em', 'margin-left': '4px'}"
+            :location="{top: true}"
+          >
```

Comment:
For FE icon, default color (Primary), no "Alert:..." needed

Reply:
Looks good:

![image](https://github.com/user-attachments/assets/83e2c8ab-d2b7-4e49-95b4-0f196eaade79)